### PR TITLE
Add more readmes/guides

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

This PR adds additional helpful guides, linked from the README. Some of the guides incorporate information that was formerly in the FAQ. I prefer having them in separate guides under headings, since those are more easily linkable than the FAQs, and I want to direct users straight to them.

I'm open for other ideas on how to organize the guides, title sections, etc. I had a hard time deciding that.

Additional guides I want to make: Local host development, Azure Architecture deep dive.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Click links from README to guides, see images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,6 +266,10 @@ By default, the deployed Azure web app will only allow requests from the same or
 
 For the frontend code, change `BACKEND_URI` in `api.ts` to point at the deployed backend URL, so that all fetch requests will be sent to the deployed backend.
 
+For an alternate frontend that's written in Web Components and deployed to Static Web Apps, check out
```
Comment (pamelafox) [inline README.md:269]: @anfibiacreativa Added shoutout to your repo here

File: docs/customization.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+### Ask approach
+
+The ask tab uses the approach programmed in [retrievethenread.py](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/app/backend/approaches/retrievethenread.py).
+
+1. It queries Azure Cognitive Search for search results for the user question (optionally using the vector embeddings for that question).
+2. It then combines the search results and user question, and asks ChatGPT API to answer the question based on the sources.
+
+The `system_chat_template` variable is currently tailored to the sample data since it starts with "You are an intelligent assistant helping Contoso Inc employees with their healthcare plan questions and employee handbook questions." Change that to match your data.
+
+## Improving answer quality
```
Comment (pamelafox) [inline docs/customization.md:45]: @charris-msft Section we talked about today

Comment (charris-msft) [reply docs/customization.md:45]: LGTM

**Review by charris-msft — COMMENTED** [review]
Looks great!
 Creating the additional docs is a great idea!

**Review by chuwik — APPROVED** [review]

**Review by mattmsft — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (anfibiacreativa) [reply README.md:269]: Thank you!

**Pull request merged**